revolv/payments/tests/models/test_payment.py

```diff
@@ -418,3 +419,89 @@ def test_admin_reinvestment(self):
         user2 = RevolvUserProfile.objects.get(pk=user2.pk)
         self.assertEquals(user1.reinvest_pool, 50.00)
         self.assertEquals(user2.reinvest_pool, 150.00)
+
+    def test_admin_reinvestment_category_preference(self):
```

Comment:
Wow, this is a quality test, great job! I think we should add another one that deals with the case when only some of one of the user's reinvest pool is depleted, and one where the pools for users who have no preferences for the project are still depleted if they're the only users who are available for reinvesting.